tests/s390x_tests/consoletest_MEMORY_chmem.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE’s openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2018 SUSE LLC
+# Copyright (C) 2018 IBM Corp.
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary:  Based on consoletest_setup.pm (console test pre setup, stopping and disabling packagekit, install curl and tar to get logs and so on)
+# modified for running the testcase MEMORY_chmem on s390x.
+
+use base "s390base";
+use testapi;
+use utils;
+use strict;
+
+sub run {
+    my $self = shift;
+    $self->copy_testsuite('MEMORY_chmem');
+    $self->execute_script('chmem.test.sh','1', 1200 );
+    $self->execute_script('checksum.chmemtest.sh','1', 300 );
+}
+
+sub post_fail_hook {
+    my $self = shift;
+#    $self->export_logs();
```

Comment:
This test fails at the moment - but I cannot see why because this line is commented

Reply:
Fixed